(2) Vox Populi/Database Changes/Civilizations/Huns.sql

```diff
@@ -92,11 +92,18 @@ VALUES
 	('IMPROVEMENT_EKI', 'YIELD_PRODUCTION', 1),
 	('IMPROVEMENT_EKI', 'YIELD_CULTURE', 1);
 
+/*
```

Comment:
I'm not removing any c++ code related to the old tables until next congress, so this will result in these improvements getting double the yields.  If you want, we can comment out the data for the new tables until you get the AI for it finished.